superset-frontend/src/views/components/MenuRight.tsx

```diff
@@ -163,13 +187,15 @@ const RightMenu = ({
                     className="data-menu"
                     title={menuIconAndLabel(menu)}
                   >
-                    {menu.childs.map(item =>
-                      typeof item !== 'string' &&
-                      item.name &&
-                      configMap[item.name] === true ? (
-                        <Menu.Item key={item.name}>
-                          <a href={item.url}> {item.label} </a>
-                        </Menu.Item>
+                    {menu.childs.map((item, idx) =>
+                      typeof item !== 'string' && item.name && item.perm ? (
+                        <>
+                          {idx === 2 && <Menu.Divider />}
+                          <Menu.Item key={item.name}>
+                            {item.url && <a href={item.url}> {item.label} </a>}
+                            {item.url === undefined && item.label}
```

Comment:
nit, but instead of two lines can you combine these by making it a ternary statement?